### ISSUE TITLE
[Feature Request] Make an option to scroll with mouse wheel without using modifier key.

### ISSUE THREAD

**Issue by generic-user** [zsviczian/excalibrain#59]
Would it be possible to add an option to use the mouse wheel to zoom in/out without modifier keys.

Thanks for this plugin, it rocks! 🚀

**Comment by zsviczian** [comment]
yes, this has been a returning request. I may just submit myself to the pressure. The reason I have not made the change (it is an easy change) is because it impacts the core Excalidraw product, and every little change I make makes it harder to keep the Obsidian plugin in sync with the developments in the core Excalidraw product. So its a trade off. If I have lots of small changes, in the end we'll lose out on the developments the excalidraw.com guys are doing.

**Comment by generic-user** [comment]
OK, I understand, maybe it can get implemented upstream? That way, the excalidraw.com guys' users can benefit too. 😺
